core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -65,6 +75,7 @@ export class JestTask extends GulpTask<IJestConfig> {
   constructor() {
     super('jest',
     {
+      cache: true,
```

Comment:
>ache: tru [](start = 7, length = 9)

no default cache folder?

Reply:
if the cache should be cleaned by gulp clean, please also override the getCleanMatch function

---
In reply to: [153653935](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153653935) [](ancestors = 153653935)

Reply:
[https://github.com/Microsoft/web-build-tools/blob/master/core-build/gulp-core-build/src/tasks/GulpTask.ts#L215](https://github.com/Microsoft/web-build-tools/blob/master/core-build/gulp-core-build/src/tasks/GulpTask.ts#L215)

---
In reply to: [153654230](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153654230) [](ancestors = 153654230,153653935)

Reply:
The default is set by Jest. 

---
In reply to: [153653935](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153653935) [](ancestors = 153653935)

Reply:
Jest default configuration is not exposed via jest-cli. It will be hard to clean it. Not doing it in this PR. 

---
In reply to: [153654230](https://github.com/Microsoft/web-build-tools/pull/453#discussion_r153654230) [](ancestors = 153654230,153653935)